dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/worker/processor/TaskKillProcessor.java

```diff
@@ -192,12 +196,13 @@ private TaskKillResponseCommand buildKillTaskResponseCommand(TaskKillRequestComm
             }
         } catch (Exception e) {
             logger.error("kill yarn job error", e);
+            throw new RuntimeException("kill yarn job error");
```

Comment:
Throwing an exception does not seem to be a good way to deal with it. Maybe it would be better if the return parameter of this method is Pair<Boolean, List<String>>